homeassistant/components/media_player/__init__.py

```diff
@@ -452,6 +452,11 @@ def media_image_url(self):
         return None
 
     @property
+    def media_image(self):
+        """Binary image, content type and hash of playing media."""
+        return (None, None, None)
```

Comment:
Problem is that the hash generated by entity_picture depends on url - all the other factors are always the same (entity_id and access token). So in reality, the hash is just a transformation of the url from an entity point-of-view, so it must be different for every image otherwise we will always generate the same url for the client. This is basically what @andrey-git commented on earlier, so I still feel certain that we need to do this to avoid caching problems. Agree?